autoreject/utils.py

```diff
@@ -108,7 +108,7 @@ def _pbar(iterable, desc, leave=True, position=None, verbose='progressbar'):
         from mne.utils import ProgressBar
 
         _ProgressBar = ProgressBar
-        if not mne.utils.check_version('mne', '0.14dev0'):
+        if mne.utils.check_version('mne', '0.14'):
```

Comment:
On my computer, I get:

```py
mne.utils.check_version('mne', '0.14')
```
as `True` and

`mne.__version__` is `0.15.dev0`. So it seems to me that you need a `not`